tests/testsuite/fixture.go

```diff
@@ -228,6 +229,25 @@ func WipeTestingInfrastructure() {
 	deployOrWipeTestingInfrastrucure(DeleteRawManifest)
 }
 
+func waitForAllDaemonSetsReady(timeout time.Duration) {
```

Comment:
This might be too aggressive. Can we focus only on our namespaces?

Reply:
I mirrored what the code waiting for pods does - it waits for pods on all namespaces to be ready

Reply:
@xpivarc 
I suppose in the tests we want all daemonsets in all namespaces to be ready. Is there a risk this might not work in some environments?

Reply:
I think there is but we can follow up on this.